cypress/integration/read-demo.spec.js

```diff
@@ -33,9 +33,6 @@ describe('Read Demo', { defaultCommandTimeout: 60000 }, () => {
       cy.get('[data-cy=secret-input-box]')
         .should('have.css', 'background-color')
         .and('contain', 'rgb(52, 53, 86)')
-      cy.get('[data-cy=submit-secret-btn]')
```

Comment:
I think this will pass if added in again now the styling on that button is fixed